src/System.Diagnostics.Debug/tests/System.Diagnostics.Debug.Tests.builds

```diff
@@ -6,7 +6,11 @@
       <OSGroup>Unix</OSGroup>
     </Project>
     <Project Include="System.Diagnostics.Debug.Tests.csproj">
-      <TestTFMs>netcore50;netcoreapp1.0;net46</TestTFMs>
```

Comment:
What prompted this split? TestTFM isn't factored into the global build options and so you are essentially building this twice in the same configuration. TestTFMs is only about which things to run.